src/dispatch.ts

```diff
@@ -113,18 +129,15 @@ export class ElectronFrontend implements CommonFrontend {
   public getOptions(): ElectronOptions {
     return this.inner.getOptions();
   }
+}
 
-  /**
-   * @inheritDoc
-   */
-  public setContext(context: Context, scope: Scope): void {
-    this.inner.setContext(context, scope);
-  }
-
-  /**
-   * @inheritDoc
-   */
-  public getInitialScope(): Scope {
-    return this.inner.getInitialScope();
-  }
+/**
+ * This either calls init on main with node interations or renderer
+ * with browser integrations.
+ * @param options Options
+ */
+export function specificInit(options: ElectronOptions): void {
+  process.type === 'browser'
+    ? module.require('./main').init(options)
+    : require('./renderer').init(options);
```

Comment:
Isn't this logic backwards? :thinking:

Reply:
It's confusing I know https://electronjs.org/docs/api/process#processtype